content/applications/marketing/email_marketing.rst

```diff
@@ -1,440 +1,817 @@
 
 Without the :guilabel:`Mailing Campaigns` feature activated, the :guilabel:`Settings` tab on the
-email detail form looks like this:
+email form only contains the :guilabel:`Preview Text`, :guilabel:`Send From`, :guilabel:`Reply To`,
+:guilabel:`Attachments`, and :guilabel:`Responsible` fields.
 
 .. image:: email_marketing/settings-without-features.png
    :align: center
-   :alt: View of settings tab in Odoo Email Marketing app, without settings activated.
+   :alt: View of settings tab in Odoo Email Marketing app, without campaign setting activated.
+
+Email content
+~~~~~~~~~~~~~
 
-- :guilabel:`Responsible`: choose an employee (in the database) to be responsible for this
-  particular email.
-- :guilabel:`Send From`: designate an email alias that'll display as the sender of this particular
+- :guilabel:`Preview Text`: allows the user to enter a preview sentence to encourage recipients to
+  open the email. In most inboxes, this is displayed next to the subject. If left empty, the first
+  characters of the email content appear, instead. The ability to add an emoji in this field is
+  available, as well, via the :guilabel:`(smiley face with a plus sign)` icon.
+- :guilabel:`Send From`: designate an email alias that displays as the sender of this particular
   email.
-- :guilabel:`Reply To`: designate an email alias to whom all the replies to this particular email
-  will be sent.
-- :guilabel:`Attachments`: if any specific documents are required (or helpful) for this event
-  invite, they can be sent along with this email, by clicking :guilabel:`ATTACH A FILE`, and adding
-  the appropriate document(s).
+- :guilabel:`Reply To`: designate an email alias to whom all the replies of this particular email
+  are sent.
+- :guilabel:`Attach a file`: if any specific files are required (or helpful) for this email, click
+  the :guilabel:`Attachments` button, and upload the desired file(s) to the email.
+
+Tracking
+~~~~~~~~
+
+- :guilabel:`Responsible`: designate a user in the database to be responsible for this particular
+  email.
+
+.. note::
+   If the :guilabel:`Mailing Campaign` feature *is* activated, an additional :guilabel:`Campaign`
+   field appears in the :guilabel:`Tracking` section of the :guilabel:`Settings` tab.
+
+   .. image:: email_marketing/settings-tab-with-campaign.png
+      :align: center
+      :alt: View of settings tab in Odoo Email Marketing when campaign setting is activated.
+
+   The additional :guilabel:`Campaign` field allows users to attach this particular email to a
+   mailing campaign, if desired.
+
+   If the desired campaign is not available in the initial drop-down menu, select :guilabel:`Search
+   More` to reveal a complete list of all mailing campaigns in the database.
+
+   Or, type the name of the desired mailing campaign in the :guilabel:`Mailing Campaign` field,
+   until Odoo reveals the desired campaign in the drop-down menu. Then, select the desired campaign.
```

Comment:
Double checking– is this supposed to be the **Campaign** field or **Mailing Campaign** field?
```suggestion
   Or, type the name of the desired mailing campaign in the :guilabel:`Campaign` field, until Odoo
   reveals the desired campaign in the drop-down menu. Then, select the desired campaign.

```